testplan/web_ui/testing/src/Common/utils.js

```diff
@@ -174,3 +175,22 @@ export const toPlainObjectIn = (obj, enumerableOnly = false) => {
 export const joinURLComponent = (base, component) => {
   return `${base.replace(/\/+$/, '').trim()}/${component}`;
 };
+
+/**
+ * Parse JSON string to object by JSON5 https://json5.org/
+ * @param {string} data - The json string
+ * @returns {object}
+ */
+export const transformResponse = (data) => {
```

Comment:
the name transformResponse is to generic, can it be called to parseToJson or something more meaningfull?